doc/source/ray-air/examples/tfx_tabular_train_to_serve.ipynb

```diff
@@ -0,0 +1,2059 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "VaFMt6AIhYbK"
+   },
+   "source": [
+    "This notebook is adapted from \"components keras\" [tutorial](https://www.tensorflow.org/tfx/tutorials/tfx/components_keras).\n",
+    "The goal is to predict whether a trip may generate a big tip.\n",
+    "\n",
+    "In this example, we showcase how to convert a tfx pipeline to [Ray AIR](https://docs.ray.io/en/latest/ray-air/getting-started.html), covering\n",
+    "every step from data ingestion to pushing a model to serving.\n",
+    "\n",
+    "1. Read a CSV file into ray dataset.\n",
+    "2. Process the dataset by chaining a variety of off-the-shelf preprocessors.\n",
+    "3. Train the model using distributed tensorflow with few lines of code.\n",
+    "4. Serve the model that will apply the same preprocessing to the incoming requests.\n",
+    "\n",
+    "Note, ``ray.ml.checkpoint.Checkpoint`` serves as the bridge between step 3 and step 4.\n",
+    "By capturing both model and preprocessing steps in a way compatible with Ray Serve, this\n",
+    "abstraction makes sure ml workload can transition seamlessly between training and\n",
+    "serving."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sQbdfyWQhYbO"
+   },
+   "source": [
+    "Uncomment and run the following line in order to install all the necessary dependencies:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "YajFzmkthYbO",
+    "outputId": "b6dd3a4f-8435-4202-9322-827222de4509"
+   },
+   "outputs": [],
+   "source": [
+    "# ! pip install \"tensorflow>=2.8.0\" \"ray[tune, data, serve]>=1.12.1\"\n",
+    "# ! ray install-nightly\n",
+    "# ! pip install fastapi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pvSRaEHChYbP"
+   },
+   "source": [
+    "# Set up Ray"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LRdL3kWBhYbQ"
+   },
+   "source": [
+    "We will use `ray.init()` to initialize a local cluster. By default, this cluster will be compromised of only the machine you are running this notebook on. You can also run this notebook on an Anyscale cluster."
```

Comment:
Remove the reference to Anyscale

Can you say something like 'use ray.init(address="auto") if you wish to connect on a cluster'